rollup.config.js

```diff
@@ -25,6 +26,7 @@ const BASE_PLUGINS = [
   nodeResolve(),
   commonjs(),
   templateLiteralIndentFix(),
+  minifyErrors(),
```

Comment:
Do we want to minify errors for all builds?

Reply:
You right! will fix